demos/projects/ST/b-l475e-iot01a/ADU.md

```diff
@@ -0,0 +1,239 @@
+You can use one of the terminal applications to monitor communication and confirm that your device is set up correctly. Go to Device Manager in Windows to determine which COM port your device was assigned.
+
+The following settings can be used to monitor serial data:
+
+- Baud Rate: `115200`
+- Data Bits: `8`
+- Stop Bits: `1`
+- Parity: none
+- Flow Control: none
+
+At some point after the device connects to Azure IoT Hub, look for this splash message:
+
+```log
+I (7165) AZ IOT: ------------------------------------------------------------------------------
+I (7165) AZ IOT: ADU SAMPLE
+I (7165) AZ IOT: Version: 1.0
+I (7165) AZ IOT: ------------------------------------------------------------------------------
+```
+
+Note the `Version: 1.0` listed.
+
+## Prepare the ADU Service
+
+To create an Azure Device Update instance and connect it to your IoT Hub, please follow the directions linked here:
+
+- [Create an Azure Device Update Instance](https://docs.microsoft.com/azure/iot-hub-device-update/create-device-update-account?tabs=portal)
+
+For other prerequisite help, please see the links below. If none of the links apply to your development environment, you may skip them.
+
+- [Create an Azure IoT Hub](https://docs.microsoft.com/azure/iot-hub/iot-hub-create-through-portal)
+- [Create Device Provisioning Service Instance](https://docs.microsoft.com/azure/iot-dps/quick-setup-auto-provision)
+
+## Deploy the Over the Air Update
+
+### Build the Update Image
+
+Modify the version of the image as below:
+
+On file `iot-middleware-freertos-samples\demos\projects\ST\b-l475e-iot01a\config\demo_config.h` ([found here](https://github.com/Azure-Samples/iot-middleware-freertos-samples/blob/vaavva/stm32l475-adu-sample/demos/projects/ST/b-l475e-iot01a/config/demo_config.h#L515))
+
+Change
+
+```c
+#define democonfigADU_UPDATE_VERSION      "1.0"
+```
+
+to
+
+```c
+#define democonfigADU_UPDATE_VERSION      "1.1"
+```
+
+The resulting binary `iot-middleware-sample-adu.bin` should be located in the build directory in `b-l475e-iot01a\demos\projects\ST\b-l475e-iot01a\`. Save it into `C:\ADU-update`, renaming it to `iot-middleware-sample-adu-v1.1.bin`.
+
+### Generate the ADU Update Manifest
+
+Navigate to the `C:\ADU-update` directory:
+
+Clone the ADU toolset.
+
+```bash
+git clone https://github.com/Azure/iot-hub-device-update
+```
+
+Generate the update manifest using **powershell**.
+
+```powershell
+Set-ExecutionPolicy -ExecutionPolicy RemoteSigned -Scope Process
+
+Import-Module .\iot-hub-device-update\tools\AduCmdlets\AduUpdate.psm1
+
+$updateId = New-AduUpdateId -Provider "STMicroelectronics" -Name "STM32L475" -Version 1.1
+
+$compat = New-AduUpdateCompatibility -Properties @{ deviceManufacturer = 'STMicroelectronics'; deviceModel = 'STM32L475' }
+
+$installStep = New-AduInstallationStep -Handler 'microsoft/swupdate:1'-HandlerProperties @{ installedCriteria = "${updateId.version}" } -Files C:\ADU-update\iot-middleware-sample-adu-v${updateId.version}.bin
```

Comment:
since this guide specifies `1.1` elsewhere
```suggestion
$installStep = New-AduInstallationStep -Handler 'microsoft/swupdate:1'-HandlerProperties @{ installedCriteria = "${updateId.version}" } -Files C:\ADU-update\iot-middleware-sample-adu-v1.1.bin
```